app.js

```diff
@@ -0,0 +1,108 @@
+// Hamburger menu
+const menu = document.querySelector('.hamburger');
+const windowMenu = document.querySelector('.window');
+
+menu.addEventListener('click', () => {
+  menu.classList.toggle('active');
+  windowMenu.classList.toggle('active');
+});
+document.querySelectorAll('.menu-link').forEach((k) => k.addEventListener('click', () => {
+  menu.classList.remove('active');
+  windowMenu.classList.remove('active');
+}));
+
+// #load content dynamically
+
+// 1.store values in an array
+
+const cardArray = [
+  {
+    img: 'https://i.pinimg.com/736x/07/33/ba/0733ba760b29378474dea0fdbcb97107.jpg',
+    name: 'Adhola Heldricks',
+    title: 'Ted Talk Enthuasist',
+    description:
+      'Give professional talks around the world about climate changes and its effects on third world nations, especialy in Africa.',
+  },
+  {
+    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRVxASP7dywthHPf5LICTwGvFmKLJCFZnF4h8QEG1pP8CbFbh1x15Cd5-j6zNIiAWz9H7g&usqp=CAU',
+    name: 'Theophalus Daku',
+    title: 'Climate columist',
+    description:
+      'climate and weather columnist at the one of the biggest african magazines that talks about environment, "AfroEnviro Paper."',
+  },
+  {
+    img: 'https://www.justheadshots.photo/wp-content/uploads/2020/04/Dark-background-headshots-015.jpg',
+    name: 'Akande Ashao',
+    title: 'CNN reporter',
+    description:
+      'covers and features areas around africa for CNN. Akande focuses on coal mining in congo and its impacts to the Congo forest.',
```

Comment:
- [OPTIONAL] Just like the previous reviewer highlighted, the image of the **_third speaker (Akande Ashao)_** is broken. When I paste the URL into the browser, it displays the site can't be reached page. I would suggest you crosscheck the URL. 👍 😎